docs/specification.md

```diff
@@ -359,8 +357,6 @@ Parameter (in Michelson):
 
 - Permission logic follows the default permissions descriptor specified in FA2.
 
-- The administrator can transfer frozen tokens from any address to any address.
```

Comment:
The logic change isn't entirely correct, perhaps my issue description is a bit ambiguous.

Currently nobody **except** the admin can transfer frozen tokens, what we should do is not allow anyone to transfer them, but rather remove the admin special permission so that nobody **including** the admin can make such transfer.

To put this in context, this issue basically reverts the change made long ago for #26 , so frozen tokens (that at the time had the id of `1`) should not be transferable at all once again.

Reply:
I see, I definitely misunderstood this. Now that I think about it, it really does not make sense to allow frozen token transfer by everyone. I'll update this in a moment.

Reply:
I've changed the logic and update the spec. Should be the correct logic now.